src/widgets/medications/medications.resource.tsx

```diff
@@ -33,17 +81,31 @@ export function fetchPatientMedications(
   patientID: string,
   status: string = "ACTIVE"
 ): Observable<PatientMedications[]> {
-  return openmrsObservableFetch(
-    `/ws/rest/v1/order?patient=${patientID}&careSetting=${CARE_SETTING}&status=${status}&asOfDate=${dayjs(
-      new Date()
-    ).format(
-      "YYYY-MM-DD"
-    )}&v=custom:(uuid,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,encounter:ref,orderer:ref,orderReason,orderType,urgency,instructions,commentToFulfiller,drug:(name,strength,concept),dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)`
+  return openmrsObservableFetch<FetchResponse<PatientMedications>>(
+    `/ws/rest/v1/order?patient=${patientID}&careSetting=${CARE_SETTING}&status=${status}&v=custom:(uuid,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,encounter:ref,orderer:ref,orderReason,orderType,urgency,instructions,commentToFulfiller,drug:(name,strength,concept),dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)`
+  ).pipe(
+    map(({ data }) => {
+      const meds: Array<PatientMedications> = [];
+      data["results"].map(result => {
+        if (result.orderType.display === DRUG_ORDER_TYPE) {
+          meds.push(result);
+        }
+      });
+      return meds;
+    })
+  );
+}
+export function fetchPatientPastMedications(
+  patientID: string,
+  status: string
+): Observable<PatientMedications[]> {
+  return openmrsObservableFetch<FetchResponse<PatientMedications>>(
```

Comment:
Same problem. It's not `openmrsObservableFetch<FetchResponse<PatientMedications>>`, but rather `openmrsObservableFetch<PatientMedications>`